aws/resource_aws_ec2_transit_gateway_peering_attachment_accepter.go

```diff
@@ -0,0 +1,205 @@
+		d.SetId("")
+		return nil
+	}
+
+	transitGatewayID := aws.StringValue(transitGatewayPeeringAttachment.AccepterTgwInfo.TransitGatewayId)
+	transitGateway, err := ec2DescribeTransitGateway(conn, transitGatewayID)
+	if err != nil {
+		return fmt.Errorf("error describing EC2 Transit Gateway (%s): %s", transitGatewayID, err)
+	}
+
+	if transitGateway.Options == nil {
+		return fmt.Errorf("error describing EC2 Transit Gateway (%s): missing options", transitGatewayID)
+	}
+
+	transitGatewayAssociationDefaultRouteTableID := aws.StringValue(transitGateway.Options.AssociationDefaultRouteTableId)
+	transitGatewayDefaultRouteTableAssociation, err := ec2DescribeTransitGatewayRouteTableAssociation(conn, transitGatewayAssociationDefaultRouteTableID, d.Id())
+	if err != nil {
+		return fmt.Errorf("error determining EC2 Transit Gateway Attachment (%s) association to Route Table (%s): %s", d.Id(), transitGatewayAssociationDefaultRouteTableID, err)
+	}
+
+	if err := d.Set("tags", tagsToMap(transitGatewayPeeringAttachment.Tags)); err != nil {
+		return fmt.Errorf("error setting tags: %s", err)
+	}
+
+	d.Set("transit_gateway_default_route_table_association", (transitGatewayDefaultRouteTableAssociation != nil))
+	d.Set("peer_account_id", (transitGatewayPeeringAttachment.AccepterTgwInfo.OwnerId != nil))
+	d.Set("peer_region", (transitGatewayPeeringAttachment.AccepterTgwInfo.Region != nil))
+	d.Set("peer_transit_gateway_id", (transitGatewayPeeringAttachment.AccepterTgwInfo.TransitGatewayId != nil))
+	d.Set("tags", (transitGatewayPeeringAttachment.Tags))
+	d.Set("transit_gateway_id", (transitGatewayPeeringAttachment.RequesterTgwInfo.TransitGatewayId))
+
+	return nil
+}
+
+func resourceAwsEc2TransitGatewayPeeringAttachmentAccepterUpdate(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).ec2conn
+
+	if d.HasChange("transit_gateway_default_route_table_association") {
+		transitGatewayID := d.Get("transit_gateway_id").(string)
+
+		transitGateway, err := ec2DescribeTransitGateway(conn, transitGatewayID)
+		if err != nil {
+			return fmt.Errorf("error describing EC2 Transit Gateway (%s): %s", transitGatewayID, err)
+		}
+
+		if transitGateway.Options == nil {
+			return fmt.Errorf("error describing EC2 Transit Gateway (%s): missing options", transitGatewayID)
+		}
+
+		if d.HasChange("transit_gateway_default_route_table_association") {
+			if err := ec2TransitGatewayRouteTableAssociationUpdate(conn, aws.StringValue(transitGateway.Options.AssociationDefaultRouteTableId), d.Id(), d.Get("transit_gateway_default_route_table_association").(bool)); err != nil {
+				return fmt.Errorf("error updating EC2 Transit Gateway Attachment (%s) Route Table (%s) association: %s", d.Id(), aws.StringValue(transitGateway.Options.AssociationDefaultRouteTableId), err)
+			}
+		}
+	}
+
+	if d.HasChange("tags") {
+		if err := setTags(conn, d); err != nil {
+			return fmt.Errorf("error updating EC2 Transit Gateway Peering Attachment (%s) tags: %s", d.Id(), err)
+		}
+	}
```

Comment:
Tagging is now handled by the shared `keyvaluetags` package, please see the [Adding Resource Tagging Support section of the Contributing Guide](https://github.com/terraform-providers/terraform-provider-aws/blob/master/.github/CONTRIBUTING.md#adding-resource-tagging-support) for more information.

```suggestion
	if d.HasChange("tags") {
		o, n := d.GetChange("tags")

		if err := keyvaluetags.Ec2UpdateTags(conn, d.Id(), o, n); err != nil {
			return fmt.Errorf("error updating EC2 Transit Gateway Peering Attachment (%s) tags: %s", d.Id(), err)
		}
	}
```